addon/components/influencer-network-modal.js

```diff
@@ -1,10 +1,34 @@
 import Component from '@ember/component';
 import { computed } from '@ember/object';
 import { GLOBAL_SUPPORT_LINK } from '@upfluence/ember-upf-utils/resources/helpdesk-links';
+import { inject as service } from '@ember/service';
 
 const INFLUENCER_NETWORK_MODAL_COOKIE = 'upf_disable_influencer_modal';
 
 export default Component.extend({
+  currentUser: service(),
+  currentUserModel: null,
```

Comment:
Same comment that the previous PR

> The name is maybe misleading, it is not really a model but a payload. Maybe call it user (currentUser name is exclude to avoid conflict with the name service)